### PR TITLE
shc: add more information link

### PR DESCRIPTION
**Pull request by mere-teresa**
See #6062

- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page description includes a link to documentation or a homepage (if applicable).

Brackets around the link, this time.
Using manned.org as source for this command.

### CODE REVIEW

File: pages/common/shc.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # shc
 
 > Generic shell script compiler.
+> More information: <https://manned.org/shc>.
```
Comment (mfrw) [inline pages/common/shc.md:4]: Wondering if we should use manned here or point towards the actual git repo, as this is not a standard command. Although, we have used `manned` earlier IMO.
Thoughts ? @navarroaxel 
```suggestion
> More information: <https://github.com/neurobin/shc>.
```

Comment (mere-teresa) [reply pages/common/shc.md:4]: I begin to be very confused with the consistency rule on these links. Maybe I did not read some documentation about some rule?
What are priorization?
* source code?
* complete documentation?
* user-friendly doc?
* accurate, up-to-date documentation?

If someone has any pointers, hints or authority on this topic, I am very interested. 😄

Comment (navarroaxel) [reply pages/common/shc.md:4]: We had this discussion a time ago #5143, IMO I'm not sure about link the man page, but that's our first rule. If there is no man page, then the Git repo is a good source. I think the man page fits for this command.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/shc.md (hunk 1 above)
Comment (mfrw) [reply pages/common/shc.md:4]: @mere-teresa I apologize, if there was a any confusion :(

**Review by mfrw — APPROVED** [review]
LGTM
Thank you for your contribution @mere-teresa

File: pages/common/shc.md (hunk 1 above)
Comment (mere-teresa) [reply pages/common/shc.md:4]: Thank you @navarroaxel 
And @mfrw it's not you, it was the context. No problem.

**Pull request merged**